### PR TITLE
Deploy `pallet-parameters` to rococo and fix dynamic_params name expand

### PR DESCRIPTION
**Pull request by ggwpez**
Changes:
- Add pallet-parameters to Rococo to configure the NIS and preimage pallet.
- Fix names of expanded dynamic params. Apparently, `to_class_case` removes suffix `s`, and `Nis` becomes `Ni` 😑. Now using `to_pascal_case`.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5802079

**Comment by ggwpez** [comment]
bot bench polkadot-pallet --pallet=pallet_parameters --runtime rococo

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5802374

File: polkadot/runtime/rococo/src/lib.rs [hunk 1]
```diff
@@ -234,6 +235,32 @@ impl PrivilegeCmp<OriginCaller> for OriginPrivilegeCmp {
...
+	use super::*;
+
+	#[dynamic_pallet_params]
+	#[codec(index = 0)]
+	pub mod nis {
+		use super::*;
+
+		/// The target of active total issuance fot the non-interactive-staking pallet.
+		#[codec(index = 0)]
+		pub static Target: Perquintill = Perquintill::zero();
```
Comment (muharem) [inline polkadot/runtime/rococo/src/lib.rs:250]: how about adding one more parameter that will have some non-zero, not none initial value? and plus we can have one parameter under root origin and another under some custom origin like staking admin.

**Review by muharem — APPROVED** [review]

File: polkadot/runtime/rococo/src/lib.rs (hunk 1 above)
Comment (ggwpez) [reply polkadot/runtime/rococo/src/lib.rs:250]: Yea good idea - just for testing all cases. Will add some more.

**Review by gupnik — APPROVED** [review]

**Pull request merged**